src/rdendian.h

```diff
@@ -111,14 +111,24 @@
 #endif
 
 #elif defined(_WIN32)
-#include <intrin.h>
+#define le16toh(x) (x)
+#define le32toh(x) (x)
+#define le64toh(x) (x)
 
+#if defined(__clang__)
+/* Clang on windows masquerades as MSVC
+ * but has strange behaviours with handling simd intrinsics
```

Comment:
Do you know of any resource/link that provides more info on the strange behaviours?

Reply:
Not a lot unfortunately:

The best I found are some breadcrumbs around the internet such as:
https://lists.llvm.org/pipermail/llvm-dev/2014-October/077367.html

When I figured out it was byte swaps I stole this:
https://chromium.googlesource.com/chromium/src/+/master/base/sys_byteorder.h#42